frontend/pages/MDMAppleSSOPage/MDMAppleSSOPage.tsx

```diff
@@ -1,19 +1,24 @@
 import React from "react";
 import { useQuery } from "react-query";
 import { AxiosError } from "axios";
+import { WithRouterProps } from "react-router";
 
-import mdmAPI from "services/entities/mdm";
+import mdmAPI, { IMDMSSOParams } from "services/entities/mdm";
 
 import SSOError from "components/MDM/SSOError";
 import Spinner from "components/Spinner/Spinner";
 import { IMdmSSOReponse } from "interfaces/mdm";
 
 const baseClass = "mdm-apple-sso-page";
 
-const DEPSSOLoginPage = () => {
-  const { error } = useQuery<void, AxiosError, IMdmSSOReponse>(
+const DEPSSOLoginPage = ({
+  location: { query },
+}: WithRouterProps<object, IMDMSSOParams>) => {
```

Comment:
nice type use with `WithRouterProps`, i'll look at using this going forward too.